core/data/init.go

```diff
@@ -95,9 +97,20 @@ func Init(conf ConfigurationStruct, l logger.LoggingClient, useConsul bool) erro
 		AddressPort: conf.ZeroMQAddressPort,
 	})
 
+	dataScrubInit()
 	return nil
 }
 
 func Destruct() {
 	dbc.CloseSession()
 }
+func dataScrubInit() {
+	//Create scrub pushed events long go routine and channel
+	scrubPushedCh = make(chan int64, 5)
```

Comment:
Define magic number `5` as constant at the top of the file.

Reply:
done